packages/suite/src/utils/suite/router.ts

```diff
@@ -1,12 +1,13 @@
 import routesConstants from '@suite-constants/routes';
+import { makePublicRouterInstance } from 'next/router';
```

Comment:
it looks like you don' actually use it anywhere

Reply:
Oh crap, fixed. Cant get autoimports from vs code work